### DISCUSSION TITLE
Is xstate fit to use in Nodejs?

### DISCUSSION THREAD

**Discussion by bennyKY** [statelyai/xstate#1549] [Q&A] [answered]
Hi, all the example that i see are in the frontend field.
Does the library suits for Nodejs?

**Comment by Andarist** [comment, marked as answer]
Yes. There is no code specific for browsers in the XState and the idea behind XState is platform/language-agnostic, so there is no problem using it with node

**Reply by bennyKY** [reply]
Thanks a lot!
